docs/content/ingestion/batch-ingestion.md

```diff
@@ -205,6 +205,7 @@ The tuningConfig is optional and default parameters will be used if no tuningCon
 |ignoreInvalidRows|Boolean|Ignore rows found to have problems.|no (default == false)|
 |useCombiner|Boolean|Use hadoop combiner to merge rows at mapper if possible.|no (default == false)|
 |jobProperties|Object|a map of properties to add to the Hadoop job configuration.|no (default == null)|
+|persistBackgroundCount|Integer|Background thread numbers to persist incremental index, use this feature there is a notable increase in memory pressure and cpu usage, but will make job finished more quickly, recommend to set to 1 if want to use this feature.|no (default == 0)|
```

Comment:
The number of new background threads to use for incremental persists. Using this feature causes a notable increase in memory pressure and cpu usage, but will make the job finish more quickly. If changing from the default of 0 (use current thread for persists), we recommend setting it to 1.